### ISSUE TITLE
ReflectorWatcher info log on eventReceived

### ISSUE THREAD

**Issue by shawkins** [fabric8io/kubernetes-client#2991]
The ReflectorWatcher.eventReceived info log https://github.com/fabric8io/kubernetes-client/blob/74cc63df9b6333d083ee24a6ff3455eaad0a6da8/kubernetes-client/src/main/java/io/fabric8/kubernetes/client/informers/cache/ReflectorWatcher.java#L51 only conveys the action.

Can this be a debug log instead?

**Comment by rohanKanojia** [comment]
Yeah, you're right. I think it creates lots of noise. Could you please raise a PR to fix it?

**Comment by shawkins** [comment]
Will do.

**Comment by akram** [comment]
I had the same remark, I disable it in my logging.properties in my app:
```
io.fabric8.kubernetes.client.informers.cache.ReflectorWatcher.level=OFF
```

**Comment by hanikesn** [comment]
IMHO trace level might even be more appropiate.

**Comment by rohanKanojia** [comment]
@hanikesn : Could you please share your review comment in the PR created?

**Comment by manusa** [comment]
Fixed in #2993

**Comment by manusa** [comment]
Relates to: #2010
